source/plugins/go/src/utils_windows.go

```diff
@@ -49,3 +50,13 @@ func EnsureGenevaOr3PNamedPipeExists(namedPipeConnection *net.Conn, datatype str
 	}
 	return true
 }
+
+func isTelegrafRunning() bool {
```

Comment:
you can make this function generic so that we can use this for other processes if required. For example

func isProcessRunning(processName string) (bool, error) {
    // Command to list all processes with the given name
    cmd := exec.Command("tasklist", "/FI", fmt.Sprintf("IMAGENAME eq %s", processName))
    output, err := cmd.Output()
    if err != nil {
        return false, err
    }

    // Check if the process name appears in the command output
    return strings.Contains(string(output), processName), nil
}